src/sdk/basic-configuration.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Basic configurations for SDK using environment variables.
+ */
+export class BasicConfiguration {
+  /**
+   * Get the default API url.
+   * @returns {string}
+   */
+  static getApiUrl(): string {
+    if (process.env.ILAND_API_URL && process.env.ILAND_API_URL !== 'undefined') {
+      return process.env.ILAND_API_URL;
+    } else {
+      return 'https://api.ilandcloud.com';
+    }
+  }
+
+  /**
+   * Get the default keycloak authorisation url.
+   * @returns {string}
+   */
+  static getKeycloakUrl(): string {
```

Comment:
Lets call it `getAuthorizationUrl()` and lets update the environment variable, etc to be generic as well

Reply:
Ok let me change the name though.